src/phoenix.js

```diff
@@ -231,6 +233,10 @@ function requireError (err) {
       config.external_apps.map(app => apps.push(app.path))
     }
 
+    // provide global config object
+    // TODO: frozen object would be great ...
+    window.phoenixConfig = config
```

Comment:
is this not somewhere in the store ?

Reply:
yes - but I need it in a plain js file(files/src/default.js) where no store is accessible.

see https://github.com/owncloud/phoenix/pull/2651#pullrequestreview-329656888

Reply:
you can declare the whole module as a function that returns the module and then pass in the store from outside into the function

Reply:
example here: https://github.com/owncloud/phoenix/blob/master/apps/files/src/helpers/collaboratorRolesDefinition.js#L13

Reply:
THX for sharing this code snippet.

never the less - the code structure will soonish be different so this should be acceptable. Objections?

Reply:
alright